packages/vaadin-combo-box/src/vaadin-combo-box-mixin.js

```diff
@@ -603,19 +601,6 @@ export const ComboBoxMixin = (subclass) =>
       // we need to flush the DOM to make sure it doesn't get flushed in the middle of _render call
       // because that will cause problems to say the least.
       flush();
-
```

Comment:
Let's consider removing the above comment about Pre P2 iron-list. 
Also, do you think we should use `flush` from the virtual list package?

Reply:
Seems that the flush here isn't required anymore. Removing it caused one test to fail because the test was using a `<dom-repeat>` for creating a `<template>` element and then expected it to render synchronously. The `flush()` here accidentally also flushed the `<dom-repeat>` in the test. Removed the flush and changed the test to wait until the next frame before the assertions.